client/src/components/CreatePost/Form/FirstSection.js

```diff
@@ -3,13 +3,21 @@ import { Input } from "antd";
 import styled from "styled-components";
 import Head from "./Head";
 import { Section } from "../StyledModal";
+import { theme } from "constants/theme";
+const { black } = theme.colors;
 
-const TitleInput = styled(Input)``;
+const TitleInput = styled(Input)`
+  font-size: 1.5rem !important;
```

Comment:
Avoid the important. Try a rule like `.ant-modal-root .ant-modal-content .ant.modal-body & {`

Reply:
Same below